annotations/models.py

```diff
@@ -771,6 +771,18 @@ class Appellation(Annotation, Interpreted):
        We now fully implement the quadruple data model, so this is no longer
        relevant.
     """
+    dateStringRep = models.CharField(max_length=30, null=True, blank=True)
+    """
+    Field to store the date when  Appellation is attached to a date string.
+    """
+    CONCEPT = 'concept'
+    DATE = 'date'
+    ACHOICES = (
```

Comment:
What is A choices? Just appellation choices in case that comment ever gets deleted.